subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -2158,7 +2158,7 @@ static int handle_request(struct zoap_packet *request,
 
 	default:
 		SYS_LOG_ERR("Unknown operation: %u", context.operation);
-		return -EINVAL;
+		r = -EINVAL;
 	}
```

Comment:
that one could be merged with the first of this patch-set too

Reply:
Done